contracts/TopShotShardedCollection.cdc

```diff
@@ -37,19 +37,20 @@
 
 import NonFungibleToken from 0xNFTADDRESS
 import TopShot from 0xTOPSHOTADDRESS
+import ViewResolver   from 0xVIEWRESOLVERADDRESS
 
-pub contract TopShotShardedCollection {
+access(all) contract TopShotShardedCollection {
 
     // ShardedCollection stores a dictionary of TopShot Collections
     // A Moment is stored in the field that corresponds to its id % numBuckets
-    pub resource ShardedCollection: TopShot.MomentCollectionPublic, NonFungibleToken.Provider, NonFungibleToken.Receiver, NonFungibleToken.CollectionPublic { 
+    access(all) resource ShardedCollection: TopShot.MomentCollectionPublic, NonFungibleToken.Provider, NonFungibleToken.Receiver, NonFungibleToken.CollectionPublic, ViewResolver.ResolverCollection {
```

Comment:
I would just use `TopeShot.MomentCollectionPublic`, and `NonFungibleToken.Collection` here